cmd/queue/main_test.go

```
@@ -443,6 +443,54 @@ func TestQueueTraceSpans(t *testing.T) {
 	}
 }
 
+func BenchmarkProxyHandler(b *testing.B) {
+	var httpHandler http.HandlerFunc = func(w http.ResponseWriter, r *http.Request) {
+		if r.Header.Get(activator.RevisionHeaderName) != "" {
+			w.WriteHeader(http.StatusBadRequest)
+			return
+		}
+		if r.Header.Get(activator.RevisionHeaderNamespace) != "" {
+			w.WriteHeader(http.StatusBadRequest)
+			return
+		}
+		if got, want := r.Host, wantHost; got != want {
+			b.Errorf("Host header = %q, want: %q", got, want)
```

Do we want this, or rather return an 5XX error?